Package.swift

```diff
@@ -3,6 +3,6 @@ import PackageDescription
 let package = Package(
     name: "LeafErrorMiddleware",
     dependencies: [
-        .Package(url: "https://github.com/vapor/vapor.git", majorVersion: 2),
+        .Package(url: "https://github.com/vapor/vapor.git", .branch("master") ),
```

Comment:
For the next version of the middleware that supports Vapor 3, it will be Swift 4 only, so this should be migrated to Swift 4 syntax and the `Package@swift-4.swift` can be removed.

Additionally we need to pin to a release rather than the `master` branch otherwise there could be breaking changes that break this package. I'm happy to pin to `3.0.0-rc` for now until Vapor 3 is out